### DISCUSSION TITLE
can Metro support notify icon/ bubble notification?

### DISCUSSION THREAD

**Discussion by li-zheng-hao** [MahApps/MahApps.Metro#4322] [Q&A] [answered]
1. Notify Icon

![image](https://user-images.githubusercontent.com/22129824/176825970-f1914f94-e116-4efd-b2aa-e1942ed24e42.png)

2. bubble notification

it can be shown at  right top position of screen 

![image](https://user-images.githubusercontent.com/22129824/176825824-1e2c7bd8-3df0-4e44-b956-898fe1ca6bd8.png)

these feature is already supported in [HandyControl](https://github.com/HandyOrg/HandyControl) , Since I am currently using metro, the styles of the two are not the same, can these functions be supported in future?

**Comment by punker76** [comment, marked as answer]
@li-zheng-hao You can also use one of these libraries which are theme aware of MahApps, so you can style it to your needs:

- https://github.com/hardcodet/wpf-notifyicon
- https://github.com/Federerer/Notifications.Wpf

**Reply by li-zheng-hao** [reply]
wow  , thanks  for your reply ! it save me a lot of time
